src/simplewallet/simplewallet.cpp

```diff
@@ -4383,7 +4383,7 @@ boost::optional<epee::wipeable_string> simple_wallet::new_wallet(const boost::pr
   }
   success_msg_writer() << "**********************************************************************";
 
-  return std::move(password);
+  return password;
```

Comment:
These should've been `return {std::move(password)};`, although a quick disassembly shows the the compiler treats it the same.